lib/stat_tracker.rb

```diff
@@ -46,10 +46,10 @@ def initialize(locations = nil)
     csv_game_files
     csv_team_files
     csv_game_team_files
-    @highest_total_score_stat  = nil
-    @lowest_total_score        = nil
-    @percentage_home_wins      = nil
-    @percentage_visitor_wins   = nil
+    @highest_total_score_stat     = nil
+    @lowest_total_score_stat      = nil
```

Comment:
Do we need to change the instance variable names? Arn't they meant to be labelled the same?

Reply:
no keep it the same for the harness can detect it